### PR TITLE
docs(recipes): add "Usage with AdminJS" recipe

### PR DESCRIPTION
**Pull request by dziraf**
Hello, I've been asked to create a recipe for usage with AdminJS. Please, let me know if this is fine and whether I have created/updated the required files. When I ran the development server, I had to duplicate my changes from `docs/docs/*` into `docs/versioned_docs/*` to be able to see them and it's the bit I'm actually confused about.

### CODE REVIEW

File: docs/static/usage-with-adminjs/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (B4nan) [inline docs/static/usage-with-adminjs/index.html:1]: please remove this file, no need for redirects when you are adding new pages

File: docs/versioned_docs/version-4.5/usage-with-adminjs.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+---
+
+## Setup
+
+Once the installation process is completed, we need to set up AdminJS endpoints and database connection.
+The process is straightforward but differs based on which `plugin` you are using. Below you can find examples specific to supported frameworks:
+
+### MikroORM + Express Plugin
+```typescript
+  import AdminJS from 'adminjs';
```
Comment (B4nan) [inline docs/versioned_docs/version-4.5/usage-with-adminjs.md:49]: no need for the initial indent, let's remove that

File: docs/versioned_docs/version-4.5/usage-with-adminjs.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+```
+
+---
+
+## Setup
+
+Once the installation process is completed, we need to set up AdminJS endpoints and database connection.
+The process is straightforward but differs based on which `plugin` you are using. Below you can find examples specific to supported frameworks:
+
+### MikroORM + Express Plugin
```
Comment (B4nan) [inline docs/versioned_docs/version-4.5/usage-with-adminjs.md:47]: please keep one empty line after each heading

**Review by B4nan — CHANGES_REQUESTED** [review]
looking good, left few minor suggestions

**Comment by dziraf** [comment]
@B4nan thank you for your comments, I've done the changes you requested

**Review by B4nan — APPROVED** [review]

**Pull request merged**